File: week_02/day_2/multiple_classes_lab/bus_lab_start_code/src/bus_stop.py
class BusStop:
    def __init__(self, name):
        self.name = name
        self.queue = []

    def queue_length(self):
        return len(self.queue)    
    
    def add_to_queue(self, person):
        self.queue.append(person)

    def clear_queue(self):
        self.queue.clear()    

File: week_12/day_2/LabBusStop/src/main/java/BusStop.java
import java.util.ArrayList;

public class BusStop {
    private String name;
    private ArrayList<Person> queue;

    public BusStop(String name){
        this.name = name;
        this.queue = new ArrayList<>();
    }

    public int getQueueLength(){
        return this.queue.size();
    }

    public void addPassenger(Person person) {
        this.queue.add(person);
    }

    public Person removePassenger() {
        return queue.remove(0);
    }
}
